### PR TITLE
Remove pyaptly and other small fixes

### PR DESCRIPTION
**Pull request by eni23**
This pull request removes pyaptly and adds packages directly with aptly-commands. It also includes other small fixes.

### CODE REVIEW

File: pkgbot/__init__.py [hunk 1]
```diff
@@ -342,7 +341,10 @@ def process_request(data):
             )
 
             # copy file to final location
-            shutil.copyfile(pkg_fullpath, copy_to)
+            try:
+                shutil.copyfile(pkg_fullpath, copy_to)
+            except IOError:
+                continue
```
Comment (hairmare) [inline pkgbot/__init__.py]: It might make sense to log this to aid in tracking down any errors from here.

File: pkgbot/__init__.py [hunk 2]
```diff
@@ -436,7 +438,7 @@ def main():
     try:
         # start an http server
         server = HTTPServer(('', conf['pkgbot']['port']), RequestHandler)
-        logger.info("Started server on port {0}".format(
+        logger.info("Started v1.1 server on port {0}".format(
```
Comment (hairmare) [inline pkgbot/__init__.py]: You might want to grab this from `pkgbot/version.py` and pass it to format.

```pycon
>>> from pkgbot.version import __version__
>>> print(__version__)
1.0.1
```

**Review by hairmare — COMMENTED** [review]
I'm suggesting some smallish improvements in this review. Lemme know if you consider them worthwhile or if you'd rather add them to a backlog.

**Comment by eni23** [comment]
like this?

**Review by hairmare — APPROVED** [review]
LGTM

**Pull request merged**